src/TEAExcelToJSON/Program.cs

```diff
@@ -107,5 +81,54 @@ List<SpreadsheetDetail> ProcessTable(DataTable table)
     return details;
 }
 
+void ReadHistoricalRates(ExcelDataSetConfiguration excelDataSetConfiguration, Dictionary<(string DistrictId, int Year), decimal> dictionary)
+{
+    using (var stream = File.Open("school-district-adopted-tax-rates.xlsx", FileMode.Open, FileAccess.Read))
+    {
+        using (var reader = ExcelReaderFactory.CreateReader(stream))
+        {
+            var result = reader.AsDataSet(excelDataSetConfiguration);
+            var table = result.Tables[0];
+            foreach (DataRow row in table.Rows)
+            {
+                var districtId = (string)row[0];
+                for (var i = 2; i < table.Columns.Count; i += 2)
+                {
+                    if (string.IsNullOrEmpty(row[i].ToString()))
+                        continue;
+
+                    var year = int.Parse(table.Columns[i].ColumnName.Trim().Substring(0, 4));
+                    var moRate = (decimal)Math.Round((double)row[i], 4);
+
+                    var isRateRaw = row[i + 1].ToString();
```

Comment:
- Maybe rename this variable and the other one below.  The prefix of 'is' made me think it was a boolean first.  I like to spell out things in variables to make it clear for readers.